src/element-stub.js

```diff
@@ -27,6 +27,11 @@ export class ElementStub extends BaseElement {
   }
 
   /** @override */
+  getPriority() {
+    throw new Error('Cannot get priority of stubbed element');
```

Comment:
@dvoytenko is it ok to just through error like this here? No need to dev.assert it?